### PR TITLE
fix: MD012/no-multiple-blanks

### PR DESCRIPTION
**Pull request by nschonni**
I've included the Find/Replace regex settings in VS Code used for both of the commits if you want to redo internally instead

### CODE REVIEW

File: docs/framework/unmanaged-api/hosting/iobjecthandle-unwrap-method.md [hunk 1]
```diff
@@ -43,4 +43,3 @@ HRESULT Unwrap (
  **.NET Framework Versions:** [!INCLUDE[net_current_v20plus](../../../../includes/net-current-v20plus-md.md)]
```
Comment (mairaw) [inline docs/framework/unmanaged-api/hosting/iobjecthandle-unwrap-method.md]:
```suggestion
```

File: docs/framework/unmanaged-api/hosting/iobjecthandle-unwrap-method.md [hunk 2]
```diff
@@ -43,4 +43,3 @@ HRESULT Unwrap (
  **.NET Framework Versions:** [!INCLUDE[net_current_v20plus](../../../../includes/net-current-v20plus-md.md)]  
   
 ## See also
```
Comment (mairaw) [inline docs/framework/unmanaged-api/hosting/iobjecthandle-unwrap-method.md]:
```suggestion
```

File: docs/framework/unmanaged-api/hosting/ivalidator-formateventinfo-method.md [hunk 3]
```diff
@@ -59,4 +59,3 @@ HRESULT FormatEventInfo(
  **.NET Framework Versions:** [!INCLUDE[net_current_v20plus](../../../../includes/net-current-v20plus-md.md)]
```
Comment (mairaw) [inline docs/framework/unmanaged-api/hosting/ivalidator-formateventinfo-method.md]:
```suggestion
```

File: docs/framework/unmanaged-api/hosting/ivalidator-formateventinfo-method.md [hunk 4]
```diff
@@ -59,4 +59,3 @@ HRESULT FormatEventInfo(
  **.NET Framework Versions:** [!INCLUDE[net_current_v20plus](../../../../includes/net-current-v20plus-md.md)]  
   
 ## See also
```
Comment (mairaw) [inline docs/framework/unmanaged-api/hosting/ivalidator-formateventinfo-method.md]:
```suggestion
```

File: docs/framework/unmanaged-api/hosting/ivalidator-validate-method.md [hunk 5]
```diff
@@ -71,4 +71,3 @@ HRESULT Validate (
  **.NET Framework Versions:** [!INCLUDE[net_current_v20plus](../../../../includes/net-current-v20plus-md.md)]
```
Comment (mairaw) [inline docs/framework/unmanaged-api/hosting/ivalidator-validate-method.md]:
```suggestion
```

File: docs/framework/unmanaged-api/hosting/ivalidator-validate-method.md [hunk 6]
```diff
@@ -71,4 +71,3 @@ HRESULT Validate (
  **.NET Framework Versions:** [!INCLUDE[net_current_v20plus](../../../../includes/net-current-v20plus-md.md)]  
   
 ## See also
```
Comment (mairaw) [inline docs/framework/unmanaged-api/hosting/ivalidator-validate-method.md]:
```suggestion
```

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 7]
```diff
@@ -434,7 +434,6 @@ The model is saved to C:\Tutorials\SentimentAnalysis\bin\Debug\netcoreapp2.1\Dat
 Sentiment: This was a very bad steak | Prediction: Negative | Probability: 0.4641322
 =============== End of Predictions ===============
 
-
```
Comment (mairaw) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: I'm afraid this might be like this in the sample @JRAlexander?

**Review by mairaw — CHANGES_REQUESTED** [review]
This is great @nschonni! Left you some comments.

File: docs/machine-learning/tutorials/sentiment-analysis.md (hunk 7 above)
Comment (nschonni) [reply docs/machine-learning/tutorials/sentiment-analysis.md]: Let me know if you want it reverted. It was just a brute force Find/Replace so this actually shouldn't be picked up by the linting inside the code fence anyway

Comment (JRAlexander) [reply docs/machine-learning/tutorials/sentiment-analysis.md]: Yes, please. Thanks, @nschonni!

Comment (nschonni) [reply docs/machine-learning/tutorials/sentiment-analysis.md]: Done

**Review by mairaw — APPROVED** [review]
Thanks @nschonni for addressing my comments.

**Pull request merged**